#goal definition
geometry_msgs/Pose goal_pose
---
#result definition
geometry_msgs/Pose result_pose
---
#feedback
geometry_msgs/Pose feedback_pose